pep/app/pods/components/modal-dialogs/user/federated-login/template.hbs

```diff
@@ -0,0 +1,33 @@
+<ModalDialog @onClose={{@onClose}} @size="md" @clickOutsideToClose={{false}} as |Modal|>
+    <Modal.header @title={{t "federatedLogin.title"}} />
+    <Modal.body>
+        <p class="text-muted">
+            {{t "federatedLogin.info"}}
+        </p>
+        <Alert
+            @isShown={{bool this.loginError}}
+            @message={{this.loginError}}
```

Comment:
doesnt look like these properties exist on this component, should this `<Alert>` be here?